src/helpers/navigation.js

```diff
@@ -15,14 +15,16 @@ export const getBackAndNext = (state, { currentPath }) => {
   if (back) {
     // TODO: Fix this hack; should work for now
     // It works because the section is always after /form/
-    const sectionName = back.fullPath.split('/')[2]
+    let sectionName = back.fullPath.split('/')[2]
```

Comment:
Had to do some weirdness here bc `package` is a reserved word in JS, and so the localization key is different. I think we can make this better at a later date.